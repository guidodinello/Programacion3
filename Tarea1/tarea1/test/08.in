test_stress 10000000
Fin