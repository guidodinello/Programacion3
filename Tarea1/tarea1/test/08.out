1>Se imprimen los primeros 100: 
1 1 1 1 1 1 2 1 1 1 2 1 1 2 1 1 1 1 2 1 1 2 1 1 1 1 3 2 3 1 1 1 1 1 2 1 1 2 1 1 1 1 3 2 3 1 1 1 1 1 2 1 1 3 2 2 2 3 1 1 2 1 1 1 1 1 2 1 1 2 1 1 1 1 3 2 3 1 1 1 1 1 2 1 1 3 2 2 2 3 1 1 2 1 1 1 1 1 2 1 
2>Fin.
